kokoro/scripts/build/build_package.ps1

```diff
@@ -47,10 +47,24 @@ PACKAGE_VERSION,$env:PKG_VERSION
   Out-File -FilePath "$env:KOKORO_ARTIFACTS_DIR/custom_sponge_config.csv" -Encoding ascii
 
 Invoke-Program git submodule update --init
-Invoke-Program docker build -t $tag -f './Dockerfile.windows' .
+$artifact_registry="us-docker.pkg.dev"
+& gcloud auth configure-docker $artifact_registry
+
+$cache_location="${artifact_registry}/stackdriver-test-143416/google-cloud-ops-agent-build-cache/ops-agent-cache:windows"
+Invoke-Program docker build --cache-from="${cache_location}" \-t $tag -f './Dockerfile.windows' .
 Invoke-Program docker create --name $name $tag
 Invoke-Program docker cp "${name}:/work/out" $env:KOKORO_ARTIFACTS_DIR
 
+
+# Tell our continuous build to update the cache. Our other builds do not
+# write to any kind of cache, for example a per-PR cache, because the
+# push takes a few minutes and adds little value over just using the continuous
+# build's cache.
+if ("$env:KOKORO_ROOT_JOB_TYPE" -eq "CONTINUOUS_INTEGRATION") {
```

Comment:
no need for quotes around KOKORO_ROOT_JOB_TYPE, and le'ts use single quotes around CONTINUOUS_INTEGRATION to comply with go/powershell-style.